src/reanimated2/frameCallback/FrameCallbackRegistryUI.ts

```diff
@@ -56,6 +96,7 @@ export const prepareUIRegistry = runOnUI(() => {
         this.runCallbacks();
       } else {
         this.frameCallbackActive.delete(frameCallbackId);
+        this.frameCallbackStartTime.delete(frameCallbackId);
```

Comment:
Maybe we should have a single `Map` with some struct instead instead of two, but that's just a suggestion.